app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -458,9 +460,10 @@ public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
                                 if(controller != null && controller.canSetPlaybackSpeed()) {
                                     float playbackSpeed = (progress + 10) / 20.0f;
                                     controller.setPlaybackSpeed(playbackSpeed);
-                                    String speed = String.format("%.2f", playbackSpeed);
-                                    UserPreferences.setPlaybackSpeed(speed);
-                                    txtvPlaybackSpeed.setText(speed + "x");
+                                    String speedPref = String.format(Locale.US, "%.2f", playbackSpeed);
```

Comment:
> add a comment on why we have to do this (it's rather unexpected)

Well, what would that comment say? ^^
Basically, that is how String.format works... and that is why there is a Locale parameter

> Couldn't UserPreferences return/set the speed as a float and then users of the interface don't have to mess with this?

That way my first thought, yes. Then I remembered that I have no idea how shared preferences are actually persisted. Are the values all strings and ``prefs.getInteger(...)`` basically reads the string and parses it?
In the worst case, this could even vary...